assets/css/fonts.css

```diff
@@ -0,0 +1,16 @@
+/* latin-ext */
+@font-face {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Roboto'), local('Roboto-Regular'), url(https://fonts.gstatic.com/s/roboto/v16/Ks_cVxiCiwUWVsFWFA3Bjn-_kf6ByYO6CLYdB4HQE-Y.woff2) format('woff2');
```

Comment:
I think we should host this ourselves, it's fine for development but now is the time to think about production.

This might be possible to do using bower/npm, or git lfs if we ever enabled that. I would prefer it if we could avoid checking in binaries in the repo, thank you very much.